cmd/up/up.go

```diff
@@ -467,10 +467,11 @@ func (up *upContext) getCurrentDeployment(ctx context.Context, autoDeploy bool)
 		return nil, false, err
 	}
 
-	if !autoDeploy {
-		if err := utils.AskIfDeploy(up.Dev.Name, up.Dev.Namespace); err != nil {
-			return nil, false, err
-		}
+	if !autoDeploy && !up.Dev.Autocreate {
+		err = errors.UserError{
+			E:    fmt.Errorf("Can't autocreate deployment"),
+			Hint: "Deploy your app first on okteto cloud or enable it using --deploy arg or setting autocreate variable to true in your manifest"}
```

Comment:
@rlamana ^^ that's the error if running `okteto up` and the deployment doesn't exist

Reply:
Here a few suggestions on how we should present this output:

1. Can we be more concise with the error? I wouldn't based it on the "autocreate" or at least I would make it clear that the deployment wasn't found first, and then maybe mention the possibility to add the autocreate in the hint. Maybe something like: `Deployment ${name} not found in namespace '$namespace'`

2. Don't mention `Okteto Cloud` since this could be any cluster, right?

3. In the hint show a clear list of options:
``` 
   - Make sure your application is deployed and ready.
   - Make sure your context is pointing to the right namespace.
   - Set the `autocreate` property in your okteto.yml manifest if you want to automatically create a new development container.
 ```

4. Maybe we could show a link to the docs (are we doing this in other parts of the cli?).

wdyt?